packages/web-components/src/label/README.md

```diff
@@ -0,0 +1,131 @@
+# Label
+
+> A label represents a caption for an item in a user interface.
+
+<br />
+
+## **Design Spec**
+
+[Link to Label Design Spec in Figma](https://www.figma.com/file/aFHbqxQlX8fetFhyYl6STb/TF---Trident-control-specs?node-id=339%3A52012)
+
+<br />
+
+## **Engineering Spec**
+
+fluent-label is a web component implementation of an HTML label element. The fluent-label several visual font sizes (small, medium, large) and font weights (regular, semibold). The fluent-label also provides appearances for required and disabled states.
+
+### Use Case
+
+Creating a simple label element with an optional info icon and optional required state
+
+<br />
+
+## Class: `Label`
+
+<br />
+
+### **Variables**
+
+<br />
+
+### **Fields**
+
+| Name       | Privacy | Type                           | Default     | Description                                                       |
+| ---------- | ------- | ------------------------------ | ----------- | ----------------------------------------------------------------- |
+| `id`       | public  | `string`                       |             | Specifies the id of label                                         |
+| `for`      | public  | `string`                       |             | Specifies the id of the form element the label should be bound to |
+| `form`     | public  | `string`                       |             | Specifies which form the label belongs to                         |
```

Comment:
Similar question here - this reflects an HTML label but I don't think we've considered the implications of how this works in custom element land. Would this need to be form associated for this to work? What are the implications of that?